2024/02-data-comm/index.qmd

```diff
@@ -20,4 +20,5 @@ In this session, we will introduce major principles of effective data visualizat
 
 **Materials:**
 
+- [Script to accompany slides](2024/02-data-comm/script.R)
```

Comment:
Does this work?  If not, might need to put it in _quarto.yml under 
```yaml
project:
  resources:
    - 2024/02-data-comm/script.R
```

Reply:
It does not work, but it does now that I added this! Thanks!